adhocracy4/polls/assets/PollResults.jsx

```diff
@@ -156,11 +134,9 @@ export default class PollResult extends React.Component {
                               <div className="poll-slider__answer">
                                 {this.isUserAnswer(slide) && <i className="fas fa-check-circle" />} {slide.answer}
                               </div>
-                              <div className="poll-slider__count">
+                              <div className={this.props.question.other_choice_answers.length > 1 ? 'poll-slider__count--spaced' : 'poll-slider__count'}>
```

Comment:
Ah, I needed a while to get that. But this only styles the number of the answer, right? So the `3/127`. And not showing the numbers when there is only one thing in the slider comes from the slider and that's the overwrite you removed? Good catch!

Reply:
this actually just moves the number to the corner when the slider automatically removes the arrows when there's only 1 slide